Makefile

```diff
@@ -1,4 +1,4 @@
-HELM_VERSION := v2.16.6
+HELM_VERSION := v2.16.7
```

Comment:
The latest `2.x` release is [v2.16.9](https://github.com/helm/helm/releases/tag/v2.16.9), should we not update to that version?

Reply:
maybe we could do that. why not! 👍 
I was just referring to 2.16.7 as we use it also with konvoy.

Reply:
```suggestion
HELM_VERSION := v2.16.9
```

Reply:
is already done, will repush in a few seconds